### ISSUE TITLE
litView exception: "invalid literal"

### ISSUE THREAD

**Issue by benjaminfjones** [GaloisInc/abcBridge#1]
I think this is a bug, but I might be misinterpreting the purpose of `litView`.

``` haskell
import qualified Data.ABC.GIA as ABC

main = do
  ABC.SomeGraph g <- ABC.newGIA
  i  <- ABC.newInput g
  lv <- ABC.litView g i  -- ok!
  x  <- ABC.newInput g
  o  <- ABC.and g i x
  lo <- ABC.litView g o  -- *** Exception: Invalid literal
```

I'm using the HEAD rev of abcBridge and GHC 7.8.3 on Mac OSX.

**Comment by robdockins** [comment]
@benjaminfjones: Quite right, this is a bug.  `litView` requires us to reverse engineer how ABC classifies its literals and obviously we haven't gotten it right.  I'll try to track this down.

**Comment by robdockins** [comment]
Fixed by b9c2d73cc430f2a54c81cde3e575e80874afd480
